src/EnergyPlus/SizingManager.cc

```diff
@@ -1582,6 +1590,17 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
                                                  4); // Pz-sum
         OutputReportPredefined::PreDefTableEntry(
             state, state.dataOutRptPredefined->pdchS62svrClD, FinalSysSizing(AirLoopNum).AirPriLoopName, state.dataSize->DBySys(AirLoopNum), 4); // D
+        // Origin of D
+        int SysSizNum = UtilityRoutines::FindItemInList(
+            FinalSysSizing(AirLoopNum).AirPriLoopName, state.dataSize->SysSizInput, &SystemSizingInputData::AirPriLoopName);
+        if (SysSizNum == 0) SysSizNum = 1; // use first when none applicable
+        if (state.dataSize->SysSizInput(SysSizNum).OccupantDiversity == AutoSize) {
+            OutputReportPredefined::PreDefTableEntry(
+                state, state.dataOutRptPredefined->pdchS62svrClDorg, FinalSysSizing(AirLoopNum).AirPriLoopName, "Calculated from schedules");
+        } else {
+            OutputReportPredefined::PreDefTableEntry(
+                state, state.dataOutRptPredefined->pdchS62svrClDorg, FinalSysSizing(AirLoopNum).AirPriLoopName, "User-specified");
+        }
```

Comment:
New output report system column that reports what the source of the Occupant Diversity is.